modules/promotions/admin-menu-items/base-promotion-template.php

```diff
@@ -79,7 +79,7 @@ public function render() {
 
 				</div>
 
-				<iframe class="e-feature-promotion_iframe" src="<?php Utils::print_unescaped_internal_string( $this->get_video_url() ); ?>" title="Elementor" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+				<iframe class="e-feature-promotion_iframe" src="<?php Utils::print_unescaped_internal_string( $this->get_video_url() ); ?>&rel=0" title="Elementor" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Can you include boder props in _promotion.scss as a fallback once frameboder="0" is removed from newer browsers?

```	
&_iframe {
		grid-column: 2 / 3;
		grid-row: 1 / 2;
		aspect-ratio: 16/9;
		width: 608px;
		border: none; 
	}
```

Reply:
Done